### PR TITLE
Fix deprecation messages in `AdminResettingController` and `AdminResettingController`

### PR DESCRIPTION
**Pull request by phansys**
## Subject
Fix deprecation messages in `AdminResettingController` and `AdminResettingController`, since currently they are making references to non existing classes.

I am targeting this branch, because these changes respect BC.

### CODE REVIEW

**Comment by jordisala1991** [comment]
There might me something wrong on tests:

```
Since symfony/phpunit-bridge 5.1: Using "@expectedDeprecation" annotations in tests is deprecated, use the "ExpectDeprecationTrait::expectDeprecation()" method instead.
```

seems like we using the old way of expecting deprecations. Deprecated 😅

**Comment by phansys** [comment]
>There might me something wrong on tests:

I've replaced those annotations. The test should be fine now.

File: tests/Controller/AdminResettingControllerTest.php [hunk 1]
```diff
@@ -20,12 +20,15 @@
...
 use Sonata\UserBundle\Action\SendEmailAction;
 use Sonata\UserBundle\Controller\AdminResettingController;
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 
 class AdminResettingControllerTest extends TestCase
 {
+    use ExpectDeprecationTrait;
```
Comment (jordisala1991) [inline tests/Controller/AdminResettingControllerTest.php:30]: I think this is only needed if you using really old versions of phpunit. on 8.5 we should be fine without it

Comment (phansys) [reply tests/Controller/AdminResettingControllerTest.php:30]: I can confirm it is required. We are using phpunit/phpunit:8.5.0:
https://github.com/sonata-project/SonataUserBundle/blob/d60215ea263d81ee2e40ff9dc2df795c6ca1d299/phpunit.xml.dist#L35

Please, see symfony/symfony#37153.

Comment (jordisala1991) [reply tests/Controller/AdminResettingControllerTest.php:30]: Oh I see, needed then, I was confused with the expectException

**Comment by VincentLanglet** [comment]
There should be no changelog, for pedantic isn't it ?

And I think two commit would be better.
- One with the deprecation message fix
- One removing the deprecated `@expectedDeprecation`. (Not sure if it was really needed btw, since we'll remove all of them in the next major ; all the other SonataBundle use this annotation, so now we're in a not-consistent situation 😅)

**Comment by phansys** [comment]
Changelog section removed, since these changes are pedantic. It was:
## Changelog






```markdown
### Fixed
- Fixed QCNs referenced by deprecation messages at `AdminResettingController` and `AdminResettingController`;
- Fixed deprecated usage of `@exptectedDeprecation` through `ExpectDeprecationTrait::expectDeprecation()`.
```

**Comment by phansys** [comment]
>There should be no changelog, for pedantic isn't it ?

Changelog removed.

>And I think two commit would be better.

Commits splitted.

**Comment by jordisala1991** [comment]
Thank you @phansys

**Pull request merged**